recipes/omnipath/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "Omnipath" %}
+{% set version = "1.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0]|lower }}/{{ name|lower }}/{{ name|lower }}-{{ version }}.tar.gz
+  sha256: 16e0dd4ede915d9650d03905f71c44168299f5556e929c17631da5bfff3a6023
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```

Only the following is required.
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```